src/sidebars/favourites/tearout-directive.js

```diff
@@ -108,9 +108,10 @@
                                         store = window.storeService.open(window.name);
                                     }
 
-                                    dragService.overAnotherInstance((overAnotherInstance) => {
+                                    dragService.overAnotherInstance('.favourites', (overAnotherInstance) => {
                                         if (overAnotherInstance) {
                                             dragService.moveToOtherInstance(scope.stock);
+                                            dragService.destroy();
                                             store.remove(scope.stock);
                                             dragService = null;
```

Comment:
I assume it was for disposing purposes? As the directive is about to destroy itself.. guess it's not needed.